r/R/dplyr-mutate.R

```diff
@@ -38,11 +38,13 @@ mutate.arrow_dplyr_query <- function(.data,
   .data <- as_adq(.data)
 
   # Restrict the cases we support for now
-  if (length(dplyr::group_vars(.data)) > 0) {
+  has_aggregations <- any(unlist(lapply(exprs, all_funs)) %in% names(agg_funcs))
```

Comment:
Currently this will return a false negative if you have a column with the same name as one of the aggregate functions. I will push an improved definition of `all_funs` that fixes this.

Reply:
Fixed in 86ac1351069a7387342245e2230239f4182f2460 and added tests in be7d9e04f3efd38b608ca88e14a1a412e1365cda that exercise the fix.

Reply:
Thanks, I had removed this same code in a previous PR, not understanding why it was needed (and no tests failed 🤷 )

Reply:
Ah, gotcha, well now there's a test that will fail 😄